src/components/photoGallery/LargeImage.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+interface Props {
```

Comment:
Should match the name of the component
```suggestion
interface LargeImageProps {
```